File: Tarea2_LeyvaBanda/Tarea2.cpp
#include <iostream>
#include <string>
using namespace std;

struct Carro {
    string marca;
    string modelo;
    int año;
};

int main() {
    Carro c;
    c.marca = "Nissan";
    c.modelo = "Versa";
    c.año = 2020;

    cout << "Carro: " << c.marca << " " << c.modelo << " (" << c.año << ")" << endl;
    return 0;
}

File: Tarea2_LeyvaBanda/Tarea2.java
class Carro {
    String marca;
    String modelo;
    int año;
}

public class Tarea2 {
    public static void main(String[] args) {
        Carro c = new Carro();
        c.marca = "Nissan";
        c.modelo = "Versa";
        c.año = 2020;

        System.out.println("Carro: " + c.marca + " " + c.modelo + " (" + c.año + ")");
    }
}
